docs/learn/explanations.md

```diff
@@ -0,0 +1,8 @@
+# NeXus Definition Language (NXDL), XML, and YAML
+The NeXus data format, described by the NeXus Definition Language (NXDL), represents a concerted effort aimed at facilitating data exchange within scientific communities, particularly among those engaged in neutron, X-ray, and muon research [Könnecke, M. et al.](https://doi.org/10.1107/S1600576714027575). The data format is also being used by the material science community under the project [NeXus-FAIRmat proposal](https://fairmat-experimental.github.io/nexus-fairmat-proposal/) supporting FAIR (Findable, Accessible, Interoperable and Reuseable) data principles in material science. The format serves as a standardized framework for both data exchange and storage.
```

Comment:
Supporting the rewriting, we must not point though to outdated links.
Therefore, use [FAIRmat-NFDI ](https://github.com/FAIRmat-NFDI/nexus_definitions) instead of the fairmat-experimental -ish

Reply:
Irrelevant why they are outdated, they are, please update the link to as you said the https://fairmat-nfdi.github.io/nexus_definitions/, comment and resolve